index.html

```diff
@@ -39,7 +39,7 @@ <h2>
         <p class="category-heading">Choose your quiz category</p>
 
         <div class="category-container">
-          <a href="#">
+          <a href="./components/rules/rules.html">
```

Comment:
If index.html is in the root directory, and the components folder is the direct child of the root. Then, there is no need to include `./` in front of `components`.

means, `./components/` === `components/`

It will save a few bytes, and will help in performance.

Reply:
Thanks, updated directory path. And the score is just there to show  that user got marks in which questions.